### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by nperez0111**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/nperez0111/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
```
Comment (nperez0111) [inline ReadMe.md]: Not Neccessary

Comment (nperez0111) [inline ReadMe.md]: !fix

Comment (nperez0111) [inline ReadMe.md]: !fixed

**Pull request merged**